src/App.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import './App.css';
 
+import { Sum } from './components/Sum';
```

Comment:
remove unnecessary line
```suggestion
import { Sum } from './components/Sum';
```